tv_apps/smart-system/js/bootstrap.js

```diff
@@ -149,6 +149,12 @@ window.addEventListener('load', function startup() {
 
   window.core = BaseModule.instantiate('Core');
   window.core && window.core.start();
+
+  window.syncStateMachine = BaseModule.instantiate('SyncStateMachine');
+  window.syncStateMachine && window.syncStateMachine.start();
+
+  window.syncManager = BaseModule.instantiate('SyncManager');
+  window.syncManager && window.syncManager.start();
```

Comment:
I was trying to load them by ```core.js``` in the first place, but later found that the patch in bug 1094759 has only been partially merged into TV in bug 1206641. Bug 1094759 implements module loading logic and we need it to enabled  ```loadWhenIdle```. The lines here load ```syncStateMachine``` and ```syncManager``` in ```window``` so that ```Base_Module``` can find them.